metals/src/main/scala/scala/meta/internal/metals/MetalsLspService.scala

```diff
@@ -2301,10 +2334,16 @@ class MetalsLspService(
       s"Connected to Build server: ${session.main.name} v${session.version}"
     )
     cancelables.add(session)
+    buildTool.foreach(
```

Comment:
Why is it needed here now?

Reply:
Probably isn't in like 99%. If you start `scala-cli` from `.bsp` `quickConnect` will connect. And in `slowConnect` we fall into `reloadAndReindex` case, so if the`bspSession` is already set and there is no checksum it will reload. But that wasn't actually causing problems.

Reply:
Not a big issue either way, but I was wondering.